core/src/main/java/hudson/model/ComputerSet.java

```diff
@@ -227,7 +227,7 @@ public void do_launchAll(StaplerRequest req, StaplerResponse rsp) throws IOExcep
      */
     @RequirePOST
     public void doUpdateNow(StaplerRequest req, StaplerResponse rsp) throws IOException, ServletException {
-        Jenkins.get().checkPermission(Jenkins.MANAGE);
+        Jenkins.get().checkAnyPermission(Jenkins.MANAGE, Computer.EXTENDED_READ);
```

Comment:
By doing so, you do not support people who are granting specific permissions to users per computer.
For example, with matrix-auth (very popular authorization strategy), you can grant to a user the EXTENDED_READ to a single computer, while forbiding them on others computers.

<details>

<summary>Screenshot</summary>

![Screenshot_2022-04-20_141039_001](https://user-images.githubusercontent.com/2662497/164227515-795c8176-0e99-44ce-b58f-13203cc4c4ea.png)

</details>

⚠️ Please note, that you can also define it to not inherit from global, meaning that someone with Computer.EXTENDED_READ will not necessarily have it granted on every computers.

Reply:
>  at least for now

A lot of code in Jenkins was done with that in mind without further support provided after that, meaning that we are breaking some useful features just to be quicker.

-0.1 from me for this approach (= not blocking, but not "happy" with it)